invoices_app/tests.py

```diff
@@ -130,6 +136,53 @@ def test_supplier_invoices_list_only_taxpayer_invoices(self):
             [taxpayer.id for taxpayer in response.context['object_list']]
         )
 
+    def test_supplier_with_2invoices_list_only_two_invoices(self):
```

Comment:
Please consider changing the test name to: `test_supplier_with_two_invoices_list_only_two_invoices`. Also, I'm wondering what's the difference between invoice one and two?

Reply:
Resolved